ads/google/a4a/utils.js

```diff
@@ -130,6 +136,9 @@ function buildAdUrl(
   const iframeDepth = iframeNestingDepth(global);
   const dtdParam = {name: 'dtd'};
   const adElement = a4a.element;
+  if (ValidAdContainerTypes.indexOf(adElement.parentElement.tagName) >= 0) {
+    queryParams.push({name: 'amp_ct', value: adElement.parentElement.tagName});
+  }
   const allQueryParams = queryParams.concat(
```

Comment:
In fact, I will just move these changes to that PR since they're so closely related, and close this one.